sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -48,6 +48,7 @@ import {
   ProgramInfoStatus,
   ApplicationExceptionStatus,
   COEStatus,
+  StudentAppealStatus,
 } from "@/types";
 import { PropType, ref, defineComponent, computed, onMounted } from "vue";
```

Comment:
I have a doubt on the background color being added. When we did the cards, I remember a discussion mentioning that background color(red | amber | green) is for the latest update to highlight it(it was an optional acceptance criteria but it was done). 
On having that in mind I was assuming that only the **request for change in progress** to be highlighted.

![image](https://user-images.githubusercontent.com/54600590/224807870-908711a4-32a1-45e0-ab35-2baf3113c738.png)

Reply:
I believe that I am following what was agreed upon during the grooming. What you mentioned was valid before the introduction of these cards. For this specific scenario, both scenarios are currently "in progress" and the background color is reflecting it.
The application does not stop and the cards for disbursement/COE are going to still progress as time goes on.
@hellolynn-tbtb FYI